conf/nd96isr_v5.conf

```diff
@@ -0,0 +1,92 @@
+# NHC Configuration File
+#
+# Lines are in the form "<hostmask>||<check>"
+# Hostmask is a glob, /regexp/, or {noderange}
+# Comments begin with '#'
+#
+# This file was automatically generated by nhc-genconf
+# Sat May 6 00:05:37 UTC 2023
+#
+
+#######################################################################
+###
+### Filesystem checks
+###
```

Comment:
For this specific section (Filesystem checks), can you just use exactly the same checks that we use for the NDv4 conf file

Reply:
that should be fine. these checks are from the nhc autogeneration utility on the ndv5 vm

Reply:
I tested using the same checks as ndv4 and some fail like these:
ERROR:  nhc:  Health check failed:  check_hw_physmem:  Actual RAM size (1961032660 kB) greater than maximum allowed (928858388 kB).
ERROR:  nhc:  Health check failed:  check_hw_ib:  No IB port mlx5_ib0:1 is ACTIVE (LinkUp 200 Gb/sec).

For those tests, I'll use ndv5 specific args.